lib/algos/list/delimiter.js

```diff
@@ -94,30 +127,12 @@ class Delimiter {
      */
     addContents(key, value) {
         if (this._reachedMaxKeys()) {
-            return false;
+            return -1;
         }
-        const tmp = JSON.parse(value);
-        this.Contents.push({
-            key,
-            value: {
-                Size: tmp['content-length'],
-                ETag: tmp['content-md5'],
-                LastModified: tmp['last-modified'],
-                Owner: {
-                    DisplayName: tmp['owner-display-name'],
-                    ID: tmp['owner-id'],
-                },
-                StorageClass: tmp['x-amz-storage-class'],
-                Initiated: tmp.initiated,
-                Initiator: tmp.initiator,
-                EventualStorageBucket: tmp.eventualStorageBucket,
-                partLocations: tmp.partLocations,
-                creationDate: tmp.creationDate,
-            },
-        });
+        this.Contents.push({ key, value });
```

Comment:
To make S3 to do the computation instead of the leader node. S3 needs to do the parsing https://github.com/scality/S3/pull/651/commits/44157881f57ddcd55e0db5b4e0aab0b8bf7c247b#diff-d1fea59d8501a89ee1ce02464ecdcd16; no metadata impact.
To make them consistent across extensions (w.r.t Basic and Versions originally).